src/databases/ADIOS2/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ IF(VISIT_PARALLEL)
     TARGET_COMPILE_DEFINITIONS(EADIOS2Database_par PRIVATE ADIOS2_USE_MPI )
     TARGET_INCLUDE_DIRECTORIES(EADIOS2Database_par PRIVATE ${ADIOS2_PAR_INCLUDE_DIR})
     TARGET_LINK_DIRECTORIES(EADIOS2Database_par PRIVATE ${ADIOS2_PAR_LIBRARY_DIR} )
-    TARGET_LINK_LIBRARIES(EADIOS2Database_par visitcommon avtpipeline_par avtdatabase_par ${ADIOS2_PAR_LIB} ${vtk_libs} )
+    TARGET_LINK_LIBRARIES(EADIOS2Database_par visitcommon avtpipeline_par avtdatabase_par ${ADIOS2_PAR_LIB} ${ADIOS2_LIB} ${vtk_libs} )
```

Comment:
This change is no longer necessary if the  serial libs are listed in the PAR_LIB var.

Reply:
Oh, I forgot to regen after changing `.xml` file